chain/client/src/test_utils.rs

```diff
@@ -1170,14 +1170,12 @@ pub fn create_chunk(
         swap(&mut chunk, &mut encoded_chunk);
         swap(&mut merkle_paths, &mut new_merkle_paths);
     }
-    if tx_root_replaced {
-        match &mut chunk {
-            EncodedShardChunk::V1(ref mut chunk) => {
-                chunk.header.height_included = 2;
-            }
-            EncodedShardChunk::V2(ref mut chunk) => {
-                *chunk.header.height_included_mut() = 2;
-            }
+    match &mut chunk {
+        EncodedShardChunk::V1(ref mut chunk) => {
+            chunk.header.height_included = next_height;
+        }
+        EncodedShardChunk::V2(ref mut chunk) => {
+            *chunk.header.height_included_mut() = next_height;
```

Comment:
```suggestion
        EncodedShardChunk::V1(chunk) => {
            chunk.header.height_included = next_height;
        }
        EncodedShardChunk::V2(chunk) => {
            *chunk.header.height_included_mut() = next_height;
```

I thing this are no longer necessary.